db/migrate/20230721123036_remove_foreign_key_and_index_from_table_name.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveForeignKeyAndIndexFromTableName < ActiveRecord::Migration[7.0]
+  def change
+    remove_foreign_key :child_table, :parent_table
+    remove_index :child_table, :parent_id
+    remove_column :child_table, :parent_id
+  end
+end
```

Comment:
- [ ] When I run "Rails db:migrate" i get the following error - 
   <img width="580" alt="image" src="https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/84156714/f2755330-0d52-483b-9963-e344b0a457d0">

And this error point to this migration file - `20230721123036_remove_foreign_key_and_index_from_table_name.rb`, and it simply means that there is no table called `child_table` or `parent_table` in your project. These names  `child_table` and `parent_table` were just placeholders and If you read this [remove_foreign_key](https://api.rubyonrails.org/v4.2/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-remove_foreign_key) documentation, you will find how to replace the placeholder words with your actual table in the project. Please fix this.

Every other thing seems to be fine. If you are not certain of what to do, please search Google first, and then if you still cannot find a way out, you can reach me via Zoom call and I will be glad to assist.